### ISSUE TITLE
Where can I find the menu?

### ISSUE THREAD

**Issue by jaclas** [cboxdoerfer/fsearch#230]
Maybe it's my ignorance, but I couldn't find a solution either in github or google.
After some update menu in FSearch disappeared:

![obraz](https://user-images.githubusercontent.com/17312717/97116732-66142580-16ff-11eb-9ec0-99fb1665fb60.png)

How can I turn it back on? I do not have access to the configuration and database refreshing.

**Comment by cboxdoerfer** [comment]
Thx, for the bug report. Hm, I'm not exactly sure what's going on here. With `Ctrl-p` you can open the preferences dialog, can you post a screenshot of it or alternatively the content of `~/.config/fsearch/fsearch.conf`?

Also how did you install FSearch? If it's from the PPA please provide the output of `apt info fsearch-trunk`, otherwise `fsearch --version`.

**Comment by jaclas** [comment]
fsearch.conf:

```
[Interface]
single_click_open=false
highlight_search_terms=true
restore_column_configuration=false
restore_sort_order=true
double_click_path=false
enable_list_tooltips=true
enable_dark_theme=false
show_menubar=true
show_statusbar=true
show_filter=true
show_search_button=true
show_base_2_units=false
action_after_file_open=0
action_after_file_open_keyboard=false
action_after_file_open_mouse=false
show_indexing_status=true
restore_window_size=false
window_width=800
window_height=600
show_listview_icons=true
show_path_column=true
show_type_column=true
show_size_column=true
show_modified_column=true
sort_ascending=false
sort_by=Size
name_column_width=250
path_column_width=250
type_column_width=100
size_column_width=75
modified_column_width=125
name_column_pos=0
path_column_pos=1
type_column_pos=2
size_column_pos=3
modified_column_pos=4

[Dialogs]
show_dialog_failed_opening=true

[Search]
search_as_you_type=true
auto_search_in_path=true
auto_match_case=true
search_in_path=false
enable_regex=false
match_case=false
hide_results_on_empty_search=true
limit_results=true
num_results=1000

[Database]
update_database_on_launch=false
exclude_hidden_files_and_folders=false
follow_symbolic_links=false
location_1=/
location_enabled_1=true
location_update_1=true
location_num_items_1=0
exclude_files=
```
```

$ apt info fsearch-trunk
Package: fsearch-trunk
Version: 20201024+r768.a1465f7~ubuntu20.04.1
Priority: optional
Section: utils
Source: fsearch
Maintainer: Christian Boxdörfer <christian.boxdoerfer@posteo.de>
Installed-Size: 489 kB
Depends: libc6 (>= 2.4), libglib2.0-0 (>= 2.41.4), libgtk-3-0 (>= 3.21.5), libpango-1.0-0 (>= 1.14.0), libpcre3
Conflicts: fsearch, fsearch-dev
Download-Size: 106 kB
APT-Manual-Installed: yes
APT-Sources: http://ppa.launchpad.net/christian-boxdoerfer/fsearch-daily/ubuntu focal/main amd64 Packages
Description: A fast file search utility
 FSearch is a fast file search utility for Unix operating systems,
 inspired by Everything Search Engine and based on GTK+3
```

I use Ubuntu 20.04 with KDE

**Comment by cboxdoerfer** [comment]
Thx, I will try to reproduce it tomorrow.

**Comment by cboxdoerfer** [comment]
I can't confirm the issue. Did you maybe add a global menubar in KDE or enabled its titlebar menu?

**Comment by jaclas** [comment]
Damn... I am so sorry! You're right, I've turned on the global menu. I only have to explain that I am new in Linux, I recently migrated from Windows and I do not know this environment well yet. Recently I've been experimenting with KDE and switching on various GUI elements. I didn't notice that this menu works this way because I have it on an automatically hiding panel.
**Once again, I am very sorry.**

And thank you very much for FSearch, on Windows I used excellent Everything and I miss a lot of its features in Linux and FSearch goes its way, which makes me very happy.
